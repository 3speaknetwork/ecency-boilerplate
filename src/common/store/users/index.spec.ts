import { initialState } from "./index";

jest.mock("../../util/local-storage", () => ({
  getByPrefix: () => {
    return [
      "eyJ1c2VybmFtZSI6InVzZXIxIiwiYWNjZXNzVG9rZW4iOiJhY2Nlc3MgdG9rZW4iLCJyZWZyZXNoVG9rZW4iOiJyZWZyZXNoIHRva2VuIiwiZXhwaXJlc0luIjoiYSBkYXRlIn0=",
      "eyJ1c2VybmFtZSI6InVzZXIyIiwiYWNjZXNzVG9rZW4iOiJhY2Nlc3MgdG9rZW4iLCJyZWZyZXNoVG9rZW4iOiJyZWZyZXNoIHRva2VuIiwiZXhwaXJlc0luIjoiYSBkYXRlIn0=",
    ];
  },
}));

let state = initialState;

it("1- default state", () => {
  expect(state).toMatchSnapshot();
});
