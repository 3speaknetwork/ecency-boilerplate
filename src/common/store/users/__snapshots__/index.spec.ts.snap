// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1- default state 1`] = `Array []`;

exports[`2- reloadAct 1`] = `
Array [
  Object {
    "accessToken": "access token",
    "expiresIn": "a date",
    "refreshToken": "refresh token",
    "username": "user1",
  },
  Object {
    "accessToken": "access token",
    "expiresIn": "a date",
    "refreshToken": "refresh token",
    "username": "user2",
  },
]
`;
